demo-ajax-inspector.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Plugin Name: Demo AJAX Inspector
+ * Description: Creates a button that outputs the AJAX calls in a given context.
+ * Version: 1.0
+ * Author: Sam Dokus
+ * Author URI: https://www.linkedin.com/in/sam-dokus/
+ * Text Domain: demo-ajax-inspector
+ */
+
+use SDOKUS\Ajax_Inspector\Dokus__Ajax__Inspector;
+
+define( 'DEMO_PLUGIN_FILE', __FILE__ );
```

Comment:
This constant is on a global context so it would either need to be prefixed, or pass a Namespace for this file and avoid the problem.